boards/ark/fmu-v6x/default.px4board

```diff
@@ -8,6 +8,7 @@ CONFIG_BOARD_SERIAL_TEL2="/dev/ttyS4"
 CONFIG_BOARD_SERIAL_TEL3="/dev/ttyS1"
 CONFIG_BOARD_SERIAL_TEL4="/dev/ttyS3"
 CONFIG_DRIVERS_ADC_BOARD_ADC=y
+CONFIG_DRIVERS_CDCACM_AUTOSTART=y
```

Comment:
> This needs to be added to nearly all boards.

I did that initially but reverted the commit 

> So if anything the module could be skipped on newer boards (like ARK v6x)

Yes and no, the other reason why I did this work is because on the ARK Jetson Carrier the VBUS for USB is driven from the Jetson. If we try to start mavlink from the startup script sercon_main will fail because VBUS is not high yet.

Reply:
@dagar I pushed a commit which adds `CONFIG_DRIVERS_CDCACM_AUTOSTART=y` to the **default.px4board** for all boards that have `CONFIG_CDCACM=y` defined in their **nsh/defconfig**. Is there anything else needed?